win32/build/phpize.js.in

```diff
@@ -229,7 +229,7 @@ C.Write(file_get_contents(PHP_DIR + "/script/config.phpize.js"));
 // Pull in code for the base detection
 modules = file_get_contents(PHP_DIR + "/script/config.w32.phpize.in");
 
-C.WriteLine("ARG_ENABLE('debug', 'Compile with debugging symbols', \"no\");");
+C.WriteLine("ARG_ENABLE('debug', 'Compile with debugging symbols', PHP_DEBUG);");

```

Comment:
I'm sorry, It has to be quoted.
I fixed it and sent a PR from another branch, please check https://github.com/php/php-src/pull/1848

Thanks,